src/core/utils/misc.cpp

```diff
@@ -241,6 +242,11 @@ int Utils::Misc::pythonVersion()
         python_proc.start("python3", QStringList() << "--version", QIODevice::ReadOnly);
         if (python_proc.waitForFinished()) {
             if (python_proc.exitCode() == 0) {
+                QByteArray output = python_proc.readAllStandardOutput();
+                if (output.isEmpty())
+                    output = python_proc.readAllStandardError();
+                const QByteArray version_str = output.split(' ').last();
+                Logger::instance()->addMessage(QCoreApplication::translate("misc", "Python version: %1").arg(QString(version_str)), Log::INFO);
                 version = 3;
```

Comment:
Before you go and write code I am wondering about "look for python executable". Can you tell me what you have in mind in terms of API calls? I actually wanted to do the same for Windows too(when python is found in PATH), but I couldn't find a qt API call.